.github/workflows/c-cpp.yml

```diff
@@ -7,12 +7,126 @@ on:
     branches: [ "main" ]
 
 jobs:
+  build-ubuntu-22_04:
+    runs-on: ubuntu-22.04
+    steps:
+    - name: Upgrade packages
+      run: |
+        sudo apt-get update
+        sudo apt-get upgrade -y
```

Comment:
That does nothing at all. It would behave exactly the same but make two simple lines into a less readable chained command.

Github Actions has `set -e` as a default bash option, so any failing command will abort the run and short-circuit just like a chain would.